clic/src/tier7.cpp

```diff
@@ -13,17 +13,28 @@
 namespace cle::tier7
 {
 
-// auto
-// affine_transform_func(const Device::Pointer & device,
-//                       const Array::Pointer &  src,
-//                       Array::Pointer          dst,
-//                       std::array<float, 16>   transform,
-//                       bool                    interpolate,
-//                       bool                    resize) -> Array::Pointer
-// {
-//   auto transform_matrix = AffineTransform(transform);
-//   return apply_affine_transform(src, dst, transform_matrix, interpolate, resize);
-// }
+auto
+eroded_otsu_labeling_func(const Device::Pointer & device,
+                          const Array::Pointer &  src,
+                          Array::Pointer          dst,
+                          int                     number_of_erosion,
```

Comment:
```suggestion
                          int                     number_of_erosions,
```
One more little thing that makes it API-compatible with the `_prototype`